README.md

```diff
@@ -909,20 +910,61 @@ can be useful for computing metrics on usage for the API.
-add a field `queryables` with the value as the JSON Schema definition of the queryables
-for that collection. This will be used for a collection's queryables resource, and removed
-from the Collection entity whenever that is returned.
+The Filter Extension defines a
+[Queryables](https://docs.ogc.org/is/19-079r2/19-079r2.html#queryables) resource for
+discovering properties that may be used to construct filter expressions. Queryables for
+each Collection are served from the `/collections/{collectionId}/queryables` endpoint.
+Root-level (global to all Collections) queryables are served from the `/queryables`
+endpoint.
+
+Collection queryables are defined in stac-server by adding a `queryables` field to the
+Collection JSON object with the value being the JSON Schema definition of the queryables
+for that Collection. The content of this `queryables` field is extracted from the
+Collection object and served from the Collection's queryables endpoint, but is removed
+from the Collection object when the Collection itself is served from the
+`/collections/{collectionId}` endpoint. Stac-server's root-level queryables resource is
+not configurable and currently does not advertise any queryable properties. Likewise, if a
+Collection does not define a `queryables` field, no queryable properties are advertised
+for that Collection. For reference, here is a queryables JSON Schema definition that does
+not advertise any queryables properties (note the empty `properties` field):
 
-A non-configurable root-level queryables definition is defined with no named terms but
-`additionalProperties` set to `true`.
+```json
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://stac-api.example.com/queryables",
+  "type": "object",
+  "title": "Queryables for Example STAC API",
+  "description": "Queryable names for the example STAC API Item Search filter.",
+  "properties": {},
+  "additionalProperties": true
+}
+```
+
+### Filter Extension
+
+Stac-server currently implements the Filter Extension such that the `id`, `collection`,
+`geometry`, and `bbox` Item fields and all fields in the `properties` object of an Item
+are always available as filter terms for a Collection, regardless of whether a Collection
+defines a `queryables` field or not. This behavior aligns with a value of `true` for the
+`additionalProperties` field in the queryables definition. Stac-server currently
+_requires_ `additionalProperties` to be `true`; a value of `false`, which would restrict
+filtering to only those `properties` defined in a Collection's queryable schema, is not
+supported and will raise an error. Thus, adding a `queryables` field to a Collection is
+informative only - it does not change the behavior of a filter.
+
+Note that when creating a filter expression that uses fields from the `properties` object
+in an Item, the fields should not be prefixed, e.g., use `eo:cloud_cover`, not
```

Comment:
in an Item, the fields **must not** be prefixed, e.g., use `eo:cloud_cover` instead of
`properties.eo:cloud_cover`, in the filter expression.

Reply:
Updated